### ISSUE TITLE
`|rein` only toggles on rather than applying status diff

### ISSUE THREAD

**Issue by sidnym-ladrut** [urbit/urbit#5862]
**Describe the bug**
Given a desk `%desk` and a list of new agent statuses `[|/& %agent] ...`, the `|rein` command in `dojo` only turns on `[& %agent]` values and turns off all other existing user agents in the `%desk` (specified or otherwise).

**To Reproduce**
Steps to reproduce the behaviour:
1. Add custom 2 agents to the `%base` desk in a fakezod (e.g. two copies of [`skeleton.hoon`](https://urbit.org/docs/userspace/gall-guide/2-agent#example), as `%agent-1` and `%agent-2`).
2. Run the command `|rein %base [& %agent-1] [& %agent-2]` and verify that both are running with `+agents %base`.
3. Run the command `|rein %base [| %agent-1]`, then run `+agents %base` and observe that both `%agent-1` and `%agent-2` are disabled.

**Expected behaviour**
The `|rein %base [| %agent-1]` should only disable `%agent-1` while `%agent-2` should continue running.

**Screenshots**
Here's the condensed `dojo` log after running the above commands in my fakezod:
```
> |rein %base [& %skeleton] [& %skeleton-2]
>=
> +agents %base
status: running   %skeleton-2
status: running   %skeleton
> |rein %base [| %skeleton]
>=
> +agents %base
status: archived  %skeleton-2
status: archived  %skeleton
```

**System (please supply the following information, if relevant):**
 - OS: Linux64
 - Vere and Urbit OS versions:  Vere v1.9
 - Base Hash: 0v1m.it118.vaueq.4a8cc.p4jcf.6jdhi.ddoq8.e5rv9.jllhh.62l80.5bv55

**Notify maintainers**
@belisarius222 
@philipcmonk

**Comment by Fang-** [comment]
Pretty sure #5627 has this as the underlying cause. Good write-up, thanks!

**Comment by sidnym-ladrut** [comment]
Yes, you're correct! `|start` uses the same `%kiln-rein` poke under the hood.

**Comment by sidnym-ladrut** [comment]
Fixed via #5863. Thanks @Fang- and @belisarius222!
